docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,27 @@
 # 4.9.0 (Unreleased)
 
 
-## New features
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy-loading set by default
+
+HTML `<img>` tags now have the `loading="lazy"` attribute added by default on image that are:
+- Inserted via the WYSIWYG editor
+- Added to a silverstripe template as an image object e.g. `$MyImage.SetWidth(300)`
```

Comment:
Is the `SetWidth(300)` part necessary here?

Reply:
Nope, have removed